deployment/proxies.rst

```diff
@@ -76,6 +76,12 @@ That's it! It's critical that you prevent traffic from all non-trusted sources.
 If you allow outside traffic, they could "spoof" their true IP address and
 other information.
 
+.. caution::
+
+    In case you are also using `CloudFront`_ or another proxy on top of your Load Balancer, calling ``$request->server->get('REMOTE_ADDR')`` as in the above example won't be enough, as it will only whitelist the first level in the chain of trust (in this case your Load Balancer). You need to make sure you also append the IP addresses or ranges of any additional proxy to the array of trusted proxies. 
```

Comment:
“The first level”. I first read this as symfony never can see any other level. That is not true. Maybe we can rephrase this?

Reply:
You're right, I did rephrase slightly. 

Maybe the issue lies with the original code example and the `// trust *all* requests` comment. It gave me the impression it would whitelist all requests no matter what.